### PR TITLE
Platforms sub-collection for games

### PR DESCRIPTION
**Pull request by cr4zy30**

### CODE REVIEW

**Review by YousefAbdelnour — CHANGES_REQUESTED** [review]
Follow the logic in my PR for countries/name/games. should have specific format and different tables than a big table with repeated field names

File: app/src/Models/GameModel.php [hunk 1]
```diff
@@ -28,4 +28,15 @@ public function getGameById($game_id): mixed
...
+
+    public function getPlatformsByGameId($game_id): array
+    {
+        $sql = <<<SQL
+        SELECT * FROM platform p, platform_game pg
+                WHERE pg.game_id = :game_id
+                    AND pg.platform_name = p.platform_name
+        SQL;
+
+        return (array) $this->paginate($sql, ["game_id" => $game_id]);
```
Comment (YousefAbdelnour) [inline app/src/Models/GameModel.php]: genre country dev info to be added

**Review by YousefAbdelnour — CHANGES_REQUESTED** [review]
same thing. check comment on code

**Comment by cr4zy30** [comment]
Updated response with information about **country**, **developer**, and **genre**:
image

**Review by YousefAbdelnour — APPROVED** [review]
Nice.

**Pull request merged**